### DISCUSSION TITLE
Test case for Azure b2c login

### DISCUSSION THREAD

**Discussion by rashdan** [wix/Detox#4009] [Q&A]
How to write test case to tap on 'Continue' button of azure b2c login.
![Simulator Screen Shot - iPhone 13 - 2023-04-13 at 10 36 25](https://user-images.githubusercontent.com/1507500/231706051-72c3263d-cc46-4b06-8fb0-17318c7fe25f.png)

I also tried below test but didn't work

```tsx
it('should show login screen after tap', async () => {
    await expect(element(by.text('Continue'))).tap();
});
```

**Comment by abilopolyi** [comment]
await system.element(by.system.label('Continue')).tap();
